src/logger/api_event.c

```diff
@@ -117,6 +119,11 @@ void process_api_event(const struct api_event * api_event, struct Serial * seria
                 api_send_button_state(serial, &api_event->data.butt_state);
                 put_crlf(serial);
                 break;
+        case ApiEventType_LoggerStatus:
```

Comment:
Every time we want to define a new event that gets broadcasted to connected devices (including telemetry to podium) we define a new event here